src/Main.java

```diff
@@ -253,4 +256,20 @@ static void initialArrays() {
     }
 
 
+    static void requestBook(String ISBN) {
+        for (int i = 0; i < bookQuantity; i++) {
+            if (books[i][0] != ISBN) {
```

Comment:
```suggestion
            if (books[i][0].equals(ISBN)) {
```

Reply:
olması daha doğru olur

Reply:
!= not equals demek degil mi?

Reply:
Evet fakat ilk önce equals olma koşulunu kontrol et  bir boolean verisi tutup kitabın bulunmama durumunda yapılması gerekenleri orda yaptır

Reply:
== veya != referansını kontrol eder içeriği kontrol etmez

Reply:
tamamdir, tesekkurler